roles/k8s_best_practices_certsuite/tasks/teardown.yml

```diff
@@ -1,14 +1,16 @@
 ---
 # Always submit these files to DCI:
-# - Configuration files - tnf_config.yml
+# - Configuration files - tnf_config.yml (or certsuite_config.yml)
 # - Execution logs - certsuite-stdout.log and certsuite.log, if they exist
 - name: Copy config and log files related to certsuite execution
+  vars:
+    kbpc_config_file_name: "{{ (kbpc_version == 'HEAD') | ternary('certsuite_config', 'tnf_config') }}"
```

Comment:
What would happen when a new release is created? Seems to me that the use of `HEAD` is a temporary patch and won't work for a new release.

Is it worth to wait?

Applies to all the ternary expressions against `kbpc_version`  and `HEAD`

Reply:
> What would happen when a new release is created? Seems to me that the use of `HEAD` is a temporary patch and won't work for a new release.

Exactly, we'll need to update this condition to also include `kbpc_version` with the latest stable version.

> Is it worth to wait?

It's the way we usually proceed and the only way to ensure backwards compatibility, so yes, up to now, unless you have a better way of handling this, this would be the solution.

Reply:
Let's go with it then, thanks